examples/js/archive.js

```diff
@@ -190,3 +190,14 @@ document.addEventListener('click', (event) => {
     map.imgGroup.addLayer(image);
   }
 });
+
+// adds toolbar to map
```

Comment:
As i mentioned I love the spirit of this re-use of the interface. But I think that on the balance it may not be worthwhile, since it involves so many changes to the core library. Let's just try making a Save button and directly calling `downloadJson()` from that button in archive.js, and focus on minimizing the buttons a bit as discussed, with shorter names and/or icons (and tooltips).